api.planx.uk/editor/findReplace.ts

```diff
@@ -136,14 +146,18 @@ const findAndReplaceInFlow = async (
       }
 
       // if matches, proceed with mutation to update flow data
-      const response = await adminClient.request(
+      const { client: $client } = getClient();
```

Comment:
Not massively fond of this syntax but it's somewhat of a solution to this issue - https://github.com/theopensystemslab/planx-new/pull/2297#discussion_r1357004769